uwblueprint/headsupguys
# [Feature] Create modules management page

## Notion ticket link

[[Dashboard] Module Management Page](https://www.notion.so/uwblueprintexecs/Task-Board-485f846ae00243599dddba2be9bcba82?p=765041c6b84f439983bda38bfe88ba66)



## Implementation description
* Implements a skeleton of the modules page (no functionality; mock data, no button click handling, etc)
* Implements a module card
* Implements a relative time format function (Yesterday, 2 days ago, 5 days ago)
* Makes tools page spacing consistent with modules (removes spacing and centering from tools)
* Adds a `styling-test` page



## Steps to test
1. Check out `/admin/dashboard/tools` and  `/admin/dashboard/modules`
2. Try substituting some of the mock data


## What should reviewers focus on?
* Styling
* A lot of things are WIP as things are merged in

## Screenshots
![image](https://user-images.githubusercontent.com/41414030/126138792-56150ea3-cc53-4bfe-9145-7b086c692f4c.png)
![image](https://user-images.githubusercontent.com/41414030/126138887-e44be588-917e-45b3-aac5-318b84b99297.png)

## Checklist
